Filter/ClassFilter.php

```diff
@@ -13,10 +13,11 @@
 
 use Sonata\AdminBundle\Datagrid\ProxyQueryInterface;
 use Sonata\CoreBundle\Form\Type\EqualType;
-use Symfony\Component\Form\Extension\Core\ChoiceList\ChoiceList;
 
 class ClassFilter extends Filter
 {
+    const OPTION_SUB_CLASSES = 'sub_classes';
```

Comment:
I used literal "sub_classes" twice. So I moved "sub_classes" to constant. It is more flexibility than use literal. If you want I do remove it.